lib/karto_sdk/include/karto_sdk/Karto.h

```diff
@@ -5859,8 +5903,16 @@ class LocalizedRangeScanWithPoints : public LocalizedRangeScan
   LocalizedRangeScanWithPoints(const LocalizedRangeScanWithPoints &);
   const LocalizedRangeScanWithPoints & operator=(const LocalizedRangeScanWithPoints &);
 
+  friend class boost::serialization::access;
+  template<class Archive>
+  void serialize(Archive & ar, const unsigned int version)
+  {
+    ar & BOOST_SERIALIZATION_NVP(m_Points);
+    ar & BOOST_SERIALIZATION_BASE_OBJECT_NVP(LocalizedRangeScan);
+  }
+
 private:
-  const PointVectorDouble m_Points;
+  PointVectorDouble m_Points;
```

Comment:
The const members are not allowed as the instances of this class can be deserialized by the boost::serialization library.